libraries/standard/http/src/http_client.c

```diff
@@ -73,7 +265,7 @@ static HTTPStatus_t _sendHttpHeaders( const HTTPTransportInterface_t * pTranspor
     if( transportStatus < 0 )
     {
         IotLogErrorWithArgs( "Error in sending the HTTP headers over the transport "
-                             "interface: Transport status %d.",
+                             "interface: Transport returnStatus %d.",
```

Comment:
If this log message is not clear, I suggest changing it to:
"Failed to send HTTP headers: Transport send() returned error: TransportStatus=%d" as it clearly mentions the _<Failure Outcome>:<Cause>:<Data>_

Reply:
I just changed that by accident. Added back the original again.